code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/IdentityExtensionTests.java

```diff
@@ -471,6 +472,153 @@ public void test_handleHubSharedState_doesNotShareStateIfLegacyECIDDoesNotChange
 			.setXDMSharedEventState(sharedStateCaptor.capture(), eq(event), any(ExtensionErrorCallback.class));
 	}
 
+	// ========================================================================================
+	// handleUrlVariablesRequest
+	// ========================================================================================
+
+	@Test
+	public void test_handleUrlVariablesRequest_nullEvent_shouldNotThrow() {
+		// test
+		extension.handleUrlVariablesRequest(null);
+	}
+
+	@Test
+	public void test_handleUrlVariablesRequest_whenConfigAndEcidNotPresent_returnsNull() {
+		// setup
+		Event event = new Event.Builder(
+			"Test event",
+			IdentityConstants.EventType.EDGE_IDENTITY,
+			IdentityConstants.EventSource.REQUEST_IDENTITY
+		)
+			.build();
```

Comment:
Can you add the "urlvariables=true" flag for completeness? Same for these other tests.